modules/cnv-configuring-vmi-live-migration-cli.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// cnv_users_guide/cnv-configuring-vmi-eviction-strategy.adoc
+
+[id="cnv-configuring-vmi-live-migration-cli_{context}"]
+= Configuring custom virtual machine instances with the `LiveMigration` 
+eviction strategy 
+
+The `LiveMigrate` eviction strategy ensures that a virtual machine instance is 
+live migrated to another node in the case of a node being placed in 
+maintenance mode, or if the `node drain` command is performed on the node. 
+
+Only custom virtual machines need to have this feature manually configured. This
+ eviction strategy is configured by default in virtual machine created from 
+common templates, or if a guest operating system is selected in either the 
+*Create Virtual Machine* or *Create Virtual Machine Template* wizards in the 
```

Comment:
This is thanks to the templates - we do not configure eviction strategy explicitly

Reply:
Sorry @suomiy, I'm not sure I understand. Can you please elaborate?

Reply:
The VM is always created from a template - so the eviction strategy is not because of the guest operating system.

Reply:
@suomiy I removed the part about guest operating systems. Does that make more sense for you?

Reply:
yes, looks ok